ui/src/app/pages/access/controlled-tier-card.tsx

```diff
@@ -0,0 +1,197 @@
+import * as React from 'react';
+
+import { AccessModule, Profile } from 'generated/fetch';
+
+import { Button } from 'app/components/buttons';
+import { FlexColumn, FlexRow } from 'app/components/flex';
+import {
+  CheckCircle,
+  ControlledTierBadge,
+  MinusCircle,
+} from 'app/components/icons';
+import { SUPPORT_EMAIL } from 'app/components/support';
+import { WithSpinnerOverlayProps } from 'app/components/with-spinner-overlay';
+import colors from 'app/styles/colors';
+import {
+  AccessTierDisplayNames,
+  AccessTierShortNames,
+} from 'app/utils/access-tiers';
+import {
+  getAccessModuleStatusByName,
+  isCompliant,
+  redirectToNiH,
+} from 'app/utils/access-utils';
+import { getCustomOrDefaultUrl } from 'app/utils/urls';
+
+import { DARPageMode, DataDetail, styles } from './data-access-requirements';
+import { Module } from './module';
+import { ModulesForAnnualRenewal } from './modules-for-annual-renewal';
+import { ModulesForInitialRegistration } from './modules-for-initial-registration';
+
+const handleClickSupportButton = (url) => () => {
+  const adjustedUrl = getCustomOrDefaultUrl(url, `mailto:${SUPPORT_EMAIL}`);
+  window.open(adjustedUrl);
+};
+
+const ctModule = AccessModule.CTCOMPLIANCETRAINING;
+
+const ControlledTierEraModule = (props: {
+  profile: Profile;
+  eligible: boolean;
+  spinnerProps: WithSpinnerOverlayProps;
+}): JSX.Element => {
+  const { profile, eligible, spinnerProps } = props;
+
+  const moduleName = AccessModule.ERACOMMONS;
+
+  const status = getAccessModuleStatusByName(profile, moduleName);
+
+  // module is not clickable if (user is ineligible for CT) or (user has completed/bypassed module already)
+  const clickable = eligible && !isCompliant(status);
+
+  return (
+    <Module
+      active={false}
+      clickable={clickable}
+      eligible={eligible}
+      moduleAction={redirectToNiH}
+      moduleName={moduleName}
+      profile={profile}
+      spinnerProps={spinnerProps}
+      status={status}
```

Comment:
Consider migrating to this pattern as part of this refactoring or in a separate PR:
```suggestion
    <Module
      {...{ clickable, eligible, moduleName, profile, spinnerProps, status }}
      active={false}
      moduleAction={redirectToNiH}
```

Reply:
I like this. I applied it to the components that were already in the PR. I will try to keep this pattern in mind going forward.